Migrations/20241216213318_InitialCreate.cs

```diff
@@ -0,0 +1,102 @@
+                        .Annotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.IdentityByDefaultColumn),
+                    Name = table.Column<string>(type: "character varying(100)", maxLength: 100, nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_Users", x => x.Id);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "Tickets",
+                columns: table => new
+                {
+                    Id = table.Column<int>(type: "integer", nullable: false)
+                        .Annotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.IdentityByDefaultColumn),
+                    UserId = table.Column<int>(type: "integer", nullable: false),
+                    TicketType = table.Column<int>(type: "integer", nullable: false),
+                    CargoWeight = table.Column<int>(type: "integer", nullable: true),
+                    CargoVolume = table.Column<int>(type: "integer", nullable: true),
+                    NumberOfSeats = table.Column<int>(type: "integer", nullable: true)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_Tickets", x => x.Id);
+                    table.ForeignKey(
+                        name: "FK_Tickets_Users_UserId",
+                        column: x => x.UserId,
+                        principalTable: "Users",
+                        principalColumn: "Id",
+                        onDelete: ReferentialAction.Cascade);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "Flights",
+                columns: table => new
+                {
+                    Id = table.Column<int>(type: "integer", nullable: false)
+                        .Annotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.IdentityByDefaultColumn),
+                    FlightType = table.Column<int>(type: "integer", nullable: false),
+                    DepartureLocation = table.Column<string>(type: "character varying(60)", maxLength: 60, nullable: false),
+                    ArrivalLocation = table.Column<string>(type: "character varying(60)", maxLength: 60, nullable: false),
+                    DepartureDate = table.Column<DateTime>(type: "timestamp with time zone", nullable: false),
+                    ArrivalDate = table.Column<DateTime>(type: "timestamp with time zone", nullable: false),
+                    BaseTicketId = table.Column<int>(type: "integer", nullable: true),
+                    CargoWeight = table.Column<int>(type: "integer", nullable: true),
+                    CargoVolume = table.Column<int>(type: "integer", nullable: true),
+                    SeatsCapacity = table.Column<int>(type: "integer", nullable: true),
+                    SeatPrice = table.Column<double>(type: "double precision", nullable: true)
```

Comment:
could we use `uint` for prices? It's usually a bad practice to use `double` for currencies, considering lower precision, no safe comparison etc

Reply:
PostgreSQL does not natively support unsigned integer types.

is int ok?